src/routes/github/github-oauth-router.ts

```diff
@@ -141,6 +141,9 @@ export const GithubAuthMiddleware = async (req: Request, res: Response, next: Ne
 	try {
 		const { query, originalUrl } = req;
 		if (query && query["resetGithubToken"]) {
+			req.session.githubToken = undefined;
+			req.session.githubRefreshToken = undefined;
```

Comment:
Curious why the tests were not failing? I think we should them test cause they don't look reliable...

Reply:
Test case added